website/sidebars.js

```diff
@@ -7,6 +7,13 @@
  * @format
  */
 
+function FBInternal(elements) {
+  return process.env.FB_INTERNAL ? elements : [];
+}
+function FBInternalWithOssFallback(elements, fallback) {
```

Comment:
Is this needed as of now, will throw a linting error on JS side if I run the linter.